extensions/amp-pan-zoom/validator-amp-pan-zoom.protoascii

```diff
@@ -44,6 +44,9 @@ tags: {  # <amp-pan-zoom>
     name: "max-scale"
     value_regex: "[0-9]+(\\.[0-9]+)?"
   }
+  attrs: {
+    name: "reset-on-resize"
```

Comment:
Should this be restricted to certain values or is it merely the presence of it does something regardless of value?

If the former then we may want to put a restrict on those values.
If the latter then it should have `value: ""` which means it shouldn't have any values.

Reply:
You're right, it shouldn't have any values. Updated.